### ISSUE TITLE
CropShapeType ellipse doesn't work with presetFixedRatioType

### ISSUE THREAD

**Issue by mschonvogel** [guoyingtao/Mantis#32]
```swift
config.presetFixedRatioType = .alwaysUsingOnePresetFixedRatio(ratio: 1)
config.cropShapeType = .ellipse
```
<img src="https://user-images.githubusercontent.com/370839/79952125-39deca00-847a-11ea-8b68-5cd7a9dbda5d.png" width=300>

**Expected:** 
<img src="https://user-images.githubusercontent.com/370839/79952071-1a47a180-847a-11ea-93cc-a28284756cbb.png" width=300>

Also, I just want the cropping UI to have this circular shape, but the actual outputted image should be a square.

**Comment by guoyingtao** [comment]
@mschonvogel Thanks for the report. I will fix this bug in the next release.
For this requirement
> Also, I just want the cropping UI to have this circular shape, but the actual outputted image should be a square.
I don't understand why you still want the actual outputted image be a square. Can you give some details about your scenario?

**Comment by guoyingtao** [comment]
I released Mantis 1.0.3 which fixed this bug.

**Comment by mschonvogel** [comment]
> I don't understand why you want the actual outputted image be a square when using a circular mask. Can you give some details about your scenario?

Sure: I'm using this for cropping profile avatars. At the moment my designer likes the avatars to be in a circular shape, but next month this could be different. This is why I have to upload the avatar as a square. (But I still want the user to be able to "see the final result")

[TOCropViewController](https://github.com/TimOliver/TOCropViewController) for instance supports this scenario by providing two delegate methods:
```swift
func cropViewController(_ cropViewController: CropViewController, didCropToCircularImage image: UIImage, withRect cropRect: CGRect, angle: Int)
```
and
```swift
func cropViewController(_ cropViewController: CropViewController, didCropToImage image: UIImage, withRect cropRect: CGRect, angle: Int)
```

**Comment by guoyingtao** [comment]
I released Mantis 1.0.4
Now you can use `config.cropShapeType = .ellipse(maskOnly: true)` to get the rect cropped image.

**Comment by mschonvogel** [comment]
@guoyingtao Awesome, thanks!
